.github/workflows/image.yml

```diff
@@ -4,8 +4,6 @@ on:
   push:
     paths:
       - "Dockerfile"
-    branches:
```

Comment:
Don't you want this to be done only when pushed on the master branch?

Reply:
We want the docker image to be updated only when we modify the dockerfile. There's no reason to re-build the image all the time